x-pack/build_chromium/windows/args.gn

```diff
@@ -18,6 +18,8 @@ use_gio = false
 use_libpci = false
 use_pulseaudio = false
 use_udev = false
+# Please, consult @elastic/kibana-security before changing/removing this option.
```

Comment:
__note:__ alternative to this comment would be adding this path to CODEOWNERS file and associating it with the security team. Comment was easier to make, but I'm open to any suggestions :)

Reply:
I think so. Really _any_ build arg changes here should probably be reviewed by the security team since the implications can be pretty wide.

Reply:
Yeah, I've added note to the docs to consult sec team before changing any of the build flags.